includes/batch.inc

```diff
@@ -241,8 +246,18 @@ function islandora_paged_content_create_ocr_batch_operation(array $pages, array
   $page = islandora_paged_content_start_batch_operation($pages, $context);
   if ($page) {
     $page = islandora_object_load($page);
-    $context['results']['successful'][] = islandora_paged_content_page_derive_ocr_datastreams($page, $options);
-    islandora_paged_content_end_batch_operation($context, 1, array('Performed OCR on @successful of @total pages.'));
+    $ocr = islandora_paged_content_page_derive_ocr_datastreams($page, $options, FALSE);
+    foreach ($ocr as $dsid => $result) {
+      if ($result !== FALSE) {
+        $context['results']['files'][] = $result;
+        // OCR files will be required for concatenation.
+        if ($dsid == 'OCR') {
+          $context['results']['pages'][] = $result;
```

Comment:
Yeah I dunno, it's kind of weird because you get OCR and HOCR and only OCR is concatenated. The pages array eventually has to be passed on to islandora_paged_content_create_ocr_paged_content_batch_operation(), so this was the easiest way I saw of making sure the two processes were separate and distinct.